src/excmds.ts

```diff
@@ -296,6 +296,44 @@ export function cssparse(...css: string[]) {
     console.log(CSS.parse(css.join(" ")))
 }
 
+/** @hidden */
+//#background
+export async function loadtheme(themename: string) {
+    if (!(await Native.nativegate("0.1.8")))
+        return
+    const separator = (await browserBg.runtime.getPlatformInfo().os) == "win" ? "\\" : "/"
+    // remove the "tridactylrc" bit so that we're left with the directory
+    const path = (await Native.getrcpath()).split(separator).slice(0, -1).join(separator)
+        + separator +  "themes" + separator + themename
+    const file = await Native.read(path)
```

Comment:
Doesn't native read handle the os specific stuff? It probably ought to...

Reply:
I have no idea, from looking at https://docs.python.org/3/library/os.path.html#os.path.expanduser it looks like it might but I doubt it handles mixed separator paths (I think findUserConfigFile will return a path with `/` on linux and a path with `\` on windows) so I can't just append something with a slash to whatever is returned.

We could try overloading the "read" command of the native messenger to accept an array on which `os.path.join` would be called though.